0009-enpoint-events-ipc/README.md

```diff
@@ -0,0 +1,126 @@
+---
+name: Endpoint events IPC
+shortname: 9/ENDPOINT_EVENTS
+status: draft
+editor: Volodymyr Tkhir <vtkhir@cybervisiontech.com>
+---
+
+* [Introduction](#introduction)
+* [Connectivity events](#connectivity-events)
+  * [Endpoint connected](#endpoint-connected)
+  * [Endpoint disconnected](#endpoint-disconnected)
+* [Lifecycle events](#lifecycle-events)
+  * [Endpoint registered](#endpoint-registered)
+  * [Endpoint updated application version](#endpoint-updated-application-version)
+* [Metadata events](#metadata-events)
+  * [Endpoint metadata updated](#endpoint-metadata-updated)
+* [Configuration events](#configuration-events)
+  * [New endpoint configuration available](#new-endpoint-configuration-available)
+  * [Endpoint applied the configuration](#endpoint-applied-the-configuration)
+
+
+## Introduction
+
+The Endpoint events IPC protocol is a [Broadcast Messaging](/0003-messaging-ipc/README.md#broadcast-messaging) protocol extension.
+
+Document describes the endpoint events that are published into NATS message broker. The document describes the NATS subject and Avro message format for each message.
+The NATS subject format is:
+
+  `kaa.v1.events.{originator-service-instance-name}.endpoint.{event-group}.{event-type}`
```

Comment:
This reminds me that there is no way to subscribe only to events from a specific application. We may want to revisit this in the future.